Automattic/wp-calypso
# Enable nav redesign on staging

Related to https://github.com/Automattic/dotcom-forge/issues/7167

## Proposed Changes

* Enable nav-redesign on staging.

## Why are these changes being made?


* To get more eyes on nav-redesign v2

## Testing Instructions



* Make sure the code looks good.
* Once merged, check that wordpress.com (with the proxy on) show the nav-redesign by default.
* Check that turning the proxy off does not show the nav-redesign.

## Pre-merge Checklist
